sources/functions/deluge

```diff
@@ -215,7 +220,12 @@ function build_deluge () {
         args="--python-disable-dependency=pyxdg --python-disable-dependency=pyopenssl -d python3-openssl -d python3-xdg"
       ;;
       1.3-stable)
-        LIST='python python-twisted python-openssl python-setuptools intltool python-xdg python-chardet python-notify python-pygame python-glade2 librsvg2-common xdg-utils python-mako'
+        if [[ $release == "focal" ]]; then
```

Comment:
Maybe turn this list the other way around?

Reply:
Can't really make assumptions about what future versions will need if they even will be able to keep compiling 1.3 so I'd rather leave it specifically for focal at this point